### ISSUE TITLE
RELEASE: Werewaffle

### ISSUE THREAD

**Issue by ohrrpgce-bugbot** [ohrrpgce/ohrrpgce#604]
[bz#604]

Hoping to make another stable release very soon.

To suggest a bug as a werewaffle blocker, just make it block this bug, or mention it in a comment here.

From: @bob-the-hamster
Reported version: 20080816 Werewaffle
Depends on:   bz⁠#577, bz⁠#603, bz⁠#611, bz⁠#616, bz⁠#624, bz⁠#630, bz⁠#631, bz⁠#635, bz⁠#637, bz⁠#639, bz⁠#644, bz⁠#650, bz⁠#653, bz⁠#666, bz⁠#671

**Comment by ohrrpgce-bugbot** [comment]
Comment author: @rversteegen

I nominate bug bz⁠#571 (plus mouse usability) for fixing (with those keyval changes, I've only tried so far to get back to the old behaviour before increasing the frame rates, ie square one)

Re: bug bz⁠#577, I tried to reproduce it at the time, but couldn't.

Also, bug bz⁠#233 is back. I've been getting very frequent freezes at startup, on that same computer which has previously inexplicitly suffered from all sorts of random crashes and this bug last time. gdb is utterly useless. Except... bug bz⁠#233 mentions it's fixed with FB0.18, but I forgot that, and I might have been using FB0.17, will have to check first. However the symptoms seem slightly different. Either way, I can't call it a blocker, I'm sure that yet again no one else will see it.

**Comment by ohrrpgce-bugbot** [comment]
Comment author: @bob-the-hamster

Nominating also bug bz⁠#603, the buy/sell defaults bug. This will require some shop code cleanup, but I am confident that the shop editor will be better for having it done.

**Comment by ohrrpgce-bugbot** [comment]
Comment author: @rversteegen

Small mention: I get "menuitem.bin record count sanity check failed 0->7" in g_debug.txt when opening Plip Tamer in Game.

**Comment by ohrrpgce-bugbot** [comment]
Comment author: @rversteegen

I just noticed: when you browse for something in Custom (besides RPG files), the colours in the browser are totally different! What's going on here? Is it using default colours unnecessary? It should really be fixed.

**Comment by ohrrpgce-bugbot** [comment]
Comment author: @bob-the-hamster

This is just a test to see if bugzilla is sending bugmail from its new server

**Comment by ohrrpgce-bugbot** [comment]
Comment author: @bob-the-hamster

\*\*\* Bug bz⁠#650 has been marked as a duplicate of this bug. \*\*\*

**Comment by ohrrpgce-bugbot** [comment]
Comment author: @bob-the-hamster

Werewaffle is released! Big thanks to everyone who participated, and most especially to TeeEmCee, who accomplished some great stuff this cycle. Yay!

**Comment by ohrrpgce-bugbot** [comment]
Comment author: @bob-the-hamster

Re-opening this bug for a werewaffle+ bugfix release

Nominating bug bz⁠#666, bug bz⁠#669, bug bz⁠#670, and bug bz⁠#671 which are all music related bugs

**Comment by ohrrpgce-bugbot** [comment]
Comment author: @bob-the-hamster

I plan on releasing werewaffle+ this afternoon

**Comment by ohrrpgce-bugbot** [comment]
Comment author: @bob-the-hamster

Werewaffle+ released more than a month ago, so I really should have re-closed this bug

(Also, testing bugmail!)
